strategy-cache-only/controlled.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Cache only: controlled page - ServiceWorker Cookbook</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+   body {
+     text-align: center;
+   }
+  </style>
+</head>
+<body>
+  <h1>Cache only</h1>
+  <img src="./asset" alt="sample asset" />
+  <p>This image request originates from a controlled page so the image will
```

Comment:
What does a "controlled page" mean? Is it a page under the service worker scope? (maybe it'd be better to describe it like that?)